pytest_html/plugin.py

```diff
@@ -33,6 +27,16 @@
 from _pytest.logging import _remove_ansi_escape_sequences
 
 
+@lru_cache()
+def ansi_support():
+    try:
+        # from ansi2html import Ansi2HTMLConverter, style  # NOQA
+        return importlib.import_module("ansi2html")
+    except ImportError:
+        # ansi2html is not installed
+        pass
```

Comment:
Looking at the usage of this function, could we have an explicit return here? Either `False` or `None`?

Reply:
Possible but not really needed. The implicit return None works really nice.